client/components/site-selector/index.jsx

```diff
@@ -68,7 +68,11 @@ export default React.createClass( {
 		this.closeSelector();
 		this.props.onSiteSelect( siteSlug );
```

Comment:
I know it's not part of this PR, so don't let this hold you up here. But it'd be nice if the behavior of site-selector was explained in its readme or in this function, that the `onSiteSelect` prop will be called with the `siteSlug`. The readme for that component should probably get beefed up in general.